tests/test_wagon.py

```diff
@@ -564,7 +490,7 @@ def setup_method(self, test_method):
         if wagon.IS_WIN:
             self.platform = 'win32'
         else:
-            self.platform = 'linux_x86_64'
```

Comment:
this is just what flask does now. Hopefully we'll stop using flask for those tests, but alas not yet